src/main/resources/com/typesafe/sbt/packager/archetypes/java_server/systemloader/upstart/start-template

```diff
@@ -27,7 +27,11 @@ end script
 # set the working directory of the job processes
 chdir ${{chdir}}
 
+setuid ${{daemon_user}}
```

Comment:
Ive tried this central is not in the sudoers file. 

Also to make this work I had to do this 
sudo chown ${{daemon_user}}:${{daemon_group}} -R  /usr/share/${{app_name}}l/